src/lib/components/consignments/__stories__/consignments-todo.story.tsx

```diff
@@ -0,0 +1,67 @@
+import { storiesOf } from "@kadira/react-native-storybook"
+import * as React from "react"
+
+import TODO from "../components/artwork-consignment-todo"
+import { ConsignmentMetadata, ConsignmentSetup } from "../index"
+
+const withArtist: ConsignmentSetup = { artist: { name: "Glenn Brown" } }
+const withOnePhoto: ConsignmentSetup = {
+  ...withArtist,
+  photos: ["https://d32dm0rphc51dk.cloudfront.net/VFiyokWNcBZNlfglZND_3g/small_square.jpg"],
+}
+
+const withPhotos: ConsignmentSetup = {
+  ...withArtist,
+  photos: [
+    "https://d32dm0rphc51dk.cloudfront.net/VFiyokWNcBZNlfglZND_3g/small_square.jpg",
+    "https://d32dm0rphc51dk.cloudfront.net/UivXcEE-GMQuBqBiHmvdcg/small_square.jpg",
+  ],
+}
+
+const metadata: ConsignmentMetadata = {
+  title: "My Work",
+  year: "1983",
+  category: "Design",
+  materials: "Wood",
+  width: 100,
+  height: 100,
+  depth: null,
+  unit: "cm",
+  displayString: "5/5",
+}
+
+const withMetadata: ConsignmentSetup = {
+  ...withPhotos,
+  metadata,
+}
+
+const withLocation: ConsignmentSetup = {
+  ...withMetadata,
+  location: "Huddersfield, UK",
+}
+
+const withProvenance: ConsignmentSetup = {
+  ...withLocation,
+  provenance: "This work has seen many hands.",
+}
+
+export const allStates = {
+  withArtist,
+  withPhotos,
+  withMetadata,
+  withLocation,
+  withProvenance,
+}
```

Comment:
These are used to generate tests - there is a thing called storyshots - https://github.com/storybooks/storybook/tree/master/addons/storyshots

It's a bit too big of a hammer for what I want ATM though.